src/core_modules/capture-core/components/WidgetRelatedStages/RelatedStagesActions/RelatedStagesActions.component.js

```diff
@@ -80,15 +84,27 @@ export const RelatedStagesActionsPlain = ({
         >
             <div className={classes.wrapper}>
                 {type === relatedStageStatus.LINKABLE ? Object.keys(mainOptionTranslatedTexts).map(key => (
-                    <Radio
-                        key={key}
-                        name={`related-stage-action-${key}`}
-                        checked={key === selectedAction}
-                        disabled={key === RelatedStagesActionTypes.LINK_EXISTING_RESPONSE && !linkableEvents.length}
-                        label={mainOptionTranslatedTexts[key]}
-                        onChange={(e: Object) => updateSelectedAction(e.value)}
-                        value={key}
-                    />
+                    <div className={classes.fieldWrapper} key={key}>
+                        <ConditionalTooltip
+                            content={i18n.t('{{ linkableStageLabel }} is not repeatable', {
+                                linkableStageLabel: programStage.stageForm.name,
+                                interpolation: { escapeValue: false },
+                            })}
+                            enabled={!isRelatedStageEligible}
+                        >
+                            <Radio
+                                name={`related-stage-action-${key}`}
+                                checked={key === selectedAction}
+                                disabled={(key === RelatedStagesActionTypes.LINK_EXISTING_RESPONSE
+                                    && !linkableEvents.length)
+                                    || !isRelatedStageEligible}
```

Comment:
In general, this line is a bit difficult to read. Perhaps you can extract it into a separate variable on line 87.

`const isDisabled = (key === ...)`

Also, the link to an existing should not be disabled if there is an event to link to.

<img width="1723" alt="image" src="https://github.com/dhis2/capture-app/assets/12842868/fd8cfac8-9dea-4bee-9059-497368f7f2be">